### PR TITLE
JSON Content Editor

### PR DESCRIPTION
**Pull request by sarahelsaig**
Content change and details in submodule https://github.com/Lombiq/Orchard-JSON-Editor/pull/54

### CODE REVIEW

**Comment by sarahelsaig** [comment]
Huh, strange. I thought this will generate a jira issue. brb!

**Comment by sarahelsaig** [comment]
The test is only red because "Add Windows Build Warning Label" fails with `Error: Parameter token or opts.auth is required`. I think it's related to this being in a fork and not to the PR itself, so please ignore it.

**Comment by Piedone** [comment]
Hehh, ironic: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/7092830642/workflow#L90 I think it trips on the unusual combination of your user being part of the Lombiq org but this coming from a fork (and it would work for a fork when opened by an outsider, when we have to approve the run first).

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -40,7 +40,7 @@
 	branch = dev
 [submodule "src/Modules/Lombiq.JsonEditor"]
 	path = src/Modules/Lombiq.JsonEditor
-	url = https://github.com/Lombiq/Orchard-JSON-Editor.git
+	url = https://github.com/sarahelsaig/Orchard-JSON-Editor.git
```
Comment (Piedone) [inline .gitmodules:43]: Note to self: revert or don't merge this.

**Pull request merged**